core/src/main/scala/caliban/GraphQL.scala

```diff
@@ -42,7 +42,7 @@ trait GraphQL[-R] { self =>
     val schemaDirectives = renderSchemaDirectives(schemaBuilder.schemaDirectives)
 
     val schema = parts.flatten.mkString("\n") match {
-      case ""        => ""
+      case ""        => s"extend schema $schemaDirectives\n"
```

Comment:
Let me know if you rather double `\n`.

Reply:
Maybe only do this when schema directives are not empty?